internal/k8s/reconciler/config/statuses.yaml

```diff
@@ -257,4 +265,4 @@
           description: >
             This reason is used with the “ResolvedRefs” condition when one of the Listener’s Routes has a BackendRef
             to an object in another namespace, where the object in the other namespace does not have a ReferencePolicy
-            explicitly allowing the reference.
\ No newline at end of file
+            explicitly allowing the reference.
```

Comment:
Do we actually use this to bubble up the error to the listener, or is this something we can rip out with the changes in this PR?

Reply:
[For future reference] this has been raised upstream at https://github.com/kubernetes-sigs/gateway-api/issues/1112#issuecomment-1099655762